src/NServiceBus.Metrics.ServiceControl/ServiceControlMonitoring.cs

```diff
@@ -255,10 +255,15 @@ protected override void OnStop()
                     {
                         Body = body,
                         MessageIntent = MessageIntentEnum.Send,
+                        TimeToBeReceived = timeToBeReceived,
```

Comment:
Just out of curiosity, why is it set here, when it is (also used as) a property on SendOptions?

Reply:
[This seems to be the answer](https://github.com/Particular/NServiceBus.Metrics.ServiceControl/pull/14#issuecomment-349960514), but I still don't understand it fully. Is this the way it was done in v5?

Reply:
I agree with @dvdstelt this is strange. Is that a bug?

Reply:
IMHO requires a code comment to clarify

Reply:
Will add comments.